sdks/python/src/opik/evaluation/models/langchain_chat_model.py

```diff
@@ -1,23 +1,34 @@
 from typing import Any
```

Comment:
we don't need this file anymore since we are not going to use langchain as a provider